app/views/cats/index.html.erb

```diff
@@ -4,7 +4,7 @@
       <div class="card-container">
       <div class="basic_info">
         <%= image_tag(cat.avatar) %>
-        <%= link_to cat.name, "/#{cat.id}" %>
+        <%= link_to cat.name, "cats/#{cat.id}" %>
```

Comment:
This link should redirect to the path instead of hard-coding the url: 
`link_to cat.name, cat_path(cat.id)`